sdk/storage/inc/datalake/datalake_options.hpp

```diff
@@ -115,14 +126,14 @@ namespace Azure { namespace Storage { namespace DataLake {
      *        operation only if the resource has been modified since the
      (        specified date and time.
     */
-    std::string IfModifiedSince;
+    Azure::Core::Nullable<std::string> IfModifiedSince;
```

Comment:
https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/storage/inc/common/access_conditions.hpp

We have `LastModifiedTimeAccessConditions` and `ETagAccessConditions`

Reply:
Integrated access condition.